.github/workflows/ci.yaml

```diff
@@ -62,13 +62,12 @@ jobs:
       - name: Download dependencies
         id: depdownload
         run: |
-          sudo apt-get update
-          sudo apt-get install -y libdevmapper-dev libbtrfs-dev libgpgme-dev
+          hack/build/ci/install-cgo-dependencies.sh
+          hack/build/ci/install-kubebuilder.sh
```

Comment:
Why was kubebuilder added here?

Reply:
Was previously installed in `./hack/build/test.sh`.
I split the file so we have a better clue what's going on.